src/__tests__/__snapshots__/LinkConnect.test.tsx.snap

```diff
@@ -84,6 +85,71 @@ exports[`LinkConnect Component renders correctly when linkToken is provided 1`]
             "https://google.com",
             "https://front-web-platform-dev",
             "https://front-b2b-api-test.azurewebsites.net",
+            "https://applink.robinhood.com",
```

Comment:
What's the reason for adding such a long list of URLs? Are all of these used in RHC flow?

Reply:
yeah, I've got all from RHC